src/server/context.ts

```diff
@@ -931,7 +931,7 @@ function toPascalCase(text: string): string {
 }
 
 function sanitizeIslandName(name: string): string {
-  const fileName = name.replace("/", "");
+  const fileName = name.replace(/\//g, "_");
```

Comment:
I'm personally against using regex when not necessary. I feel like is a lot easier to read.
```suggestion
  const fileName = name.replaceAll("/", "_");
```

Reply:
I'm neutral on this one. Either way works.